### DISCUSSION TITLE
Vite Multi-page App

### DISCUSSION THREAD

**Discussion by carloschneider** [vite-pwa/vite-plugin-pwa#704] [Q&A]
Does this plugin support [Vite Multi-page App](https://vitejs.dev/guide/build#multi-page-app)?

**Comment by userquin** [comment]
why not?

**Reply by carloschneider** [reply]
How can I generate, for example, manifest for each html file using the plugin?

**Reply by userquin** [reply]
no makes sense, the app is a multi page app, you will have multiple html pages (entry points) not multiple apps, the sw will be installed in the app

**Reply by userquin** [reply]
if you are using multi page app to create a multi app, then you can declare the pwa multiple time using distinct configuration per html entry point, you will need to install the manifest manually

**Reply by userquin** [reply]
if that's the case, then you need to register also the sw per scope, you will need to create folders, you can only register 1 sw in the app per scope, and so you need to add distinct scopes

**Reply by carloschneider** [reply]
Yes, but its something possible to do without the plugin. I'm able to create manually multiple manifests files for each html pages. Thats why I asked if the plugin support multi pages.

With the possibility to create multiple HTMLs it would be nice, and useful, to have an option to create multiples manifests.

But thank you :smile:

**Reply by userquin** [reply]
as i mention, you can add the pwa plugin multiple times, ofc, you will need to change the src dir, output dir, and sw name, you're going to create multiple sw
